tb_spaceinvaders/src/space_invaders.rs

```diff
@@ -858,21 +860,17 @@ impl toybox_core::Simulation for SpaceInvaders {
 
 impl toybox_core::State for State {
     fn lives(&self) -> i32 {
-        // If we have removed all enemies, set lives to 0 and end.
```

Comment:
Oh. This was a hack to end after one level?

Reply:
yes